### PR TITLE
adds color_output macro to enable/disable colored output in a block

### PR DESCRIPTION
**Pull request by ssfrr**

### CODE REVIEW

File: src/Suppressor.jl [hunk 1]
```diff
@@ -109,4 +136,30 @@ macro capture_err(block)
...
+## Example
+
+@color_output false begin
+    output = @capture_err begin
+        warn("should get captured, not printed")
+    end
+end
+@test output == "WARNING: should get captured, not printed\n"
+"""
+macro color_output(enabled, block)
```
Comment (Ismael-VC) [inline src/Suppressor.jl]: Should signature be `macro color_output(enabled::Bool, block)` too?

**Comment by Ismael-VC** [comment]
Escape codes are tricky business (are these platform dependent?):

* https://travis-ci.org/Ismael-VC/Suppressor.jl/jobs/288716340#L473

```julia
Expression: output == "\e[1m\e[33mWARNING: \e[39m\e[22m\e[33mshould get captured, not printed\e[39m\n"
    Evaluated: "\e[1m\e[31mWARNING: should get captured, not printed\e[0m\n" == "\e[1m\e[33mWARNING: \e[39m\e[22m\e[33mshould get captured, not printed\e[39m\n"
```

**Comment by Ismael-VC** [comment]
Thank you very much for improving Suppressor Mr. Spencer!

File: src/Suppressor.jl (hunk 1 above)
Comment (ssfrr) [reply src/Suppressor.jl]: yeah, probably a good idea. fixed.

**Comment by ssfrr** [comment]
aw, crap. AFAIK the escape codes aren't platform-specific, but they can change with Julia version. I put the codes here that I got on 0.6, but 0.5 is different.

Actually, i just looked at the travis config and it looks like the package isn't currently being tested on 0.6.

What julia versions do you want to support? With most of my packages I've been pretty much focusing on making sure they work on 0.6, but dropping 0.5 and earlier, and not paying much attention to 0.7 yet.

**Comment by Ismael-VC** [comment]
I want to support latest stable, I'll update the testing versions, this will require a new tag release.

**Comment by ssfrr** [comment]
Sounds good. If you want to update travis config I can rebase on top of that change.

**Comment by Ismael-VC** [comment]
I'm updating versions in #17

**Comment by Ismael-VC** [comment]
@ssfrr I've merged #17, please rebase your PR against master, after merging I'll tag a new release.

**Comment by ssfrr** [comment]
sorry for the delay on this. It's now rebased and I tweaked the test so it passes on 0.5. Looks like there's an unrelated failure on MacOS nightlies, but otherwise I think it's good.

**Comment by Ismael-VC** [comment]
Thank you for this PR, I've restarted the job, and merged! :tada:

**Pull request merged**